ethapi/block_cert_json_test.go

```diff
@@ -90,3 +92,93 @@ func TestBlockCertificateToJson(t *testing.T) {
 	require.Equal(bitset, json.Signers)
 	require.Equal(sig, json.Signature)
 }
+
+func TestBlockCertificate_JsonEncodingMatchesExpectedFormat(t *testing.T) {
+	tests := map[string]cert.BlockCertificate{
+		"empty":     makeEmptyBlockCert(),
+		"non-empty": makeTestBlockCert(t),
+	}
+
+	for name, cert := range tests {
+		t.Run(name, func(t *testing.T) {
+			validateBlockCertJsonFormat(t, cert)
```

Comment:
style consideration: since this function is only used once, it can be inlined here without risking to much readability issues ...